libraries/Microsoft.Bot.Builder/BotFrameworkAdapter.cs

```diff
@@ -1299,6 +1301,11 @@ public virtual async Task CreateConversationAsync(string channelId, string servi
             }
         }
 
+        Task<IConnectorClient> IConnectorClientBuilder.CreateConnectorClientAsync(string serviceUrl, ClaimsIdentity claimsIdentity, string audience, CancellationToken cancellationToken)
```

Comment:
> Task<IConnectorClient> IConnectorClientBuilder.CreateConnectorClient [](start = 7, length = 69)

IS this a public method?  Protected method?  

Missing comments

Reply:
I'll add the comments. It's a public method that can only be seen on IConnectorclientBuilder instances. If you have a BotFrameworkAdapter, intellisense will not show this method (though casting and calling will work). Since it is public, I'll add docs.